docs/book/src/reference/project-config.md

```diff
@@ -14,6 +14,7 @@ Following is an example of a PROJECT config file which is the result of a projec
 # and allow the plugins properly work.
 # More info: https://book.kubebuilder.io/reference/project-config.html
 domain: testproject.org
+kubebuilderVersion: v3.11.1
```

Comment:
should we try to automatically bump this for each release?

Or at least add next release version as the example? Since that will be the first version to have this new field

Reply:
*v4.5.1

Reply:
Could we add, as an example, any 4.x value since it will not be available in the 3x releases? ( example: v4.6.0 )